README.md

```diff
@@ -117,78 +131,81 @@ ORDER BY profit_amount DESC
 LIMIT 10;
 ```
 
-Postgres tip: Enter `\x` to enter "Explanded display" mode which looks nicer for results with many columns
-
-## Contributing
-
-### Guide
-
-✨ Coming soon
-
-### Pre-commit
-
-We use pre-commit to maintain a consistent style, prevent errors, and ensure test coverage.
-
-To set up, install dependencies through poetry
-```
-poetry install
-```
-
-Then install pre-commit hooks with
-```
-poetry run pre-commit install
-```
-
-### Tests
-
-Run tests with
-```
-kubectl exec deploy/mev-inspect-deployment -- poetry run pytest --cov=mev_inspect tests
-```
+Postgres tip: Enter `\x` to enter "Explanded display" mode which looks nicer for results with many columns.
 
 ## FAQ
```

Comment:
I suggest to move the FAQ and most of the usage sections to the github repository wiki.